### PR TITLE
Converted some height validations to use cChunkDef::IsValidHeight()

### PR DESCRIPTION
**Pull request by SphinxC0re**

### CODE REVIEW

**Comment by SphinxC0re** [comment]
Any objections? @cuberite/core-team

**Comment by NiLSPACE** [comment]
Nope :)

File: src/ChunkMap.cpp [hunk 1]
```diff
@@ -1859,7 +1859,7 @@ bool cChunkMap::ForEachEntityInBox(const cBoundingBox & a_Box, cEntityCallback &
 void cChunkMap::DoExplosionAt(double a_ExplosionSize, double a_BlockX, double a_BlockY, double a_BlockZ, cVector3iArray & a_BlocksAffected)
 {
 	// Don't explode if outside of Y range (prevents the following test running into unallocated memory):
-	if ((a_BlockY < 0) || (a_BlockY > cChunkDef::Height - 1))
+	if (!cChunkDef::IsValidHeight(static_cast<int>(a_BlockY)))
```
Comment (madmaxoft) [inline src/ChunkMap.cpp]: This should have been a `FloorC` instead of a cast.

Comment (SphinxC0re) [inline src/ChunkMap.cpp]: Fixed in 9d07be62d3d989f846b13e81273fd710aaf513b3

**Pull request merged**